### ISSUE TITLE
Open eds container on linux

### ISSUE THREAD

**Issue by blretribution** [sovworks/edslite#43]
it is posible to open an .eds container file in linux?

**Comment by rafaelazvdo** [comment]
Yes, if you use the Veracrypt container format, you can open and mount the encrypted file using Veracrypt software on both Windows and Linux.
